config/configure-secrets.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/python
```

Comment:
What's the rationale for adding a dependency on Python?

Reply:
Simpler yaml file parsing modules.